app/controllers/pensions/PensionsSummaryController.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.pensions
+
+import config.AppConfig
+import controllers.predicates.AuthorisedAction
+import play.api.Logging
+import play.api.i18n.I18nSupport
+import play.api.mvc._
+import services.PensionSessionService
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import views.html.pensions.PensionsSummaryView
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class PensionsSummaryController @Inject()(
+                                           appConfig: AppConfig,
```

Comment:
could make this 'implicit val appConfig: AppConfig' and then remove it from line 40